### DISCUSSION TITLE
What am I charged for? Cannot figure out

### DISCUSSION THREAD

**Discussion by vitkorbut** [community/community#28808] [Other Feature Feedback, Questions, & Ideas]
Received an email from Github: "This email is to inform you that we cannot bill your account."
But my account have always been free. What am I being charged for? Cannot figure out

**Comment by nlist2** [comment]
Potentially co-pilot if you had the free version (?)

**Reply by nlist2** [reply]
Or Github pro if you were previously using it as a student!

**Reply by vitkorbut** [reply]
Yes, that's probably it. Thank you. My student privileges stopped 7-8 years ago or so, but apparently Github decided to start charging me this month for my personal account. Weird and sneaky.  🙏

**Comment by radar-blip** [comment]
Sounds accurate to me "We cannot bill you because your account is free" .
Probably just auto-generated, I get something similar from a company I never use for anything.

**Reply by vitkorbut** [reply]
Github auto-upgraded me to Pro account without my consent.  🙏

**Reply by radar-blip** [reply]
For some reason, that doesnt surprise me.
